docs/docs/python-sdk/reference/config.mdx

```diff
@@ -152,15 +152,6 @@ The following settings can be defined in the `Config` class
 **Default value**: False<br />
 **Environment variable**: `INFRAHUB_RETRY_ON_FAILURE`<br />
 <!-- vale off -->
-## schema_converge_timeout
-
-**Property**: schema_converge_timeout<br />
-<!-- vale on -->
-**Description**: Number of seconds to wait for schema to have converged<br />
-**Type**: `integer`<br />
-**Default value**: 60<br />
-**Environment variable**: `INFRAHUB_SCHEMA_CONVERGE_TIMEOUT`<br />
-<!-- vale off -->
```

Comment:
This part shouldn't be removed. Looks like the pipeline to validate the generated docs didn't run which would indicate that the job is configured incorrectly (i.e. we should run it when the doc changes too). Removing this will cause the next job to fail. I'm guessing that you might have an old commit for the SDK submodule.